### DISCUSSION TITLE
Snippets for the terminal

### DISCUSSION THREAD

**Discussion by phirestalker** [Ylianst/MeshCentral#6219] [Ideas]
It would be great if we could enter frequently used commands into the terminal area somewhere. I'm not sure about the form the interface would take, but provide some way to pop up the list of commands (maybe searchable) and click to paste it into the terminal.

**Comment by si458** [comment]
already a feature request for this - https://github.com/Ylianst/MeshCentral/issues/5608
stemed from here https://github.com/Ylianst/MeshCentral/discussions/5606
